src/Language/Coq/Pretty.hs

```diff
@@ -121,6 +121,7 @@ ppTerm p e =
     Sort s ->
       ppSort s
     Var x ->
+      parensIf (p > PrecApp && head x == '@') $
```

Comment:
I'd like to see this predicate `head x == '@'` factored out into a separate function of type `Ident -> Bool`, which would probably belong in the AST module. Considering #30, we should try to minimize the code that depends on how `Ident` is implemented.

Reply:
Well, I'll go one step farther, and suggest that identifiers should not begin with `'@'`, and `translateIdentWithArgs` should not be prepending `'@'` to identifiers. Instead, that is just a hack to represent the notion that an identifier needs to be explicit in all its arguments, which should really be a different syntactic form in the AST. I'll implement and test this now.

Reply:
Ok, this is fixed with 4b2d7e09ecbbecb4bbdb296758ac962911d4ba3d